### DISCUSSION TITLE
Organize the each collection filter listing items in required sequence

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#11935] [Feature Requests]
It is better to have a feature to organize the filter items in the required sequence. It is annoying when there are many fields in one collection which are not required for filtering.
![unknown (6)](https://user-images.githubusercontent.com/36895131/156607985-a325cd71-8cc0-4eeb-93c9-c30d5a842ad1.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
